src/internal.c

```diff
@@ -1353,6 +1352,46 @@ int IdentifyAsn1Key(const byte* in, word32 inSz, int isPrivate, void* heap)
 
 
 #ifndef WOLFSSH_NO_RSA
+
+#ifdef HAVE_WC_RSAPRIVATEKEYDECODERAW
```

Comment:
This is not a portable way to detect this... meaning someone not using configure won't detect or use this new API. But its probably still the best option for now.

Reply:
Switched this to checking that wolfSSL is greater than v5.7.0. The new function isn't available in v5.7.0.